src/http/lib/pubsub.ts

```diff
@@ -23,30 +23,34 @@ export default class DaprPubSub {
     return res.status;
   }
 
-  async subscribe(pubSubName: string, topic: string, cb: TypeDaprPubSub) {
+  async subscribe(subs: TypeElementOfDaprPubSub[]) {
     const server = await WebServerSingleton.getServer();
+    if (subs.length < 1) return;
```

Comment:
Not required, forEach won't run on empty array, just make sure to add subs: TYPE = []